Description: Replace st call with stterm
 This patch replaces st with stterm which is the name
 for st binary on Debian system.
 .
Author: Vasudev Kamath <kamathvasudev@gmail.com>
Bug-Debian: http://bugs.debian.org/683851
Forwarded: not-needed
Last-Update: 2013-03-10

--- a/config.def.h
+++ b/config.def.h
@@ -35,7 +35,7 @@ static Bool hidebackground  = FALSE;
 /* DOWNLOAD(URI, referer) */
 #define DOWNLOAD(d, r) { \
 	.v = (char *[]){ "/bin/sh", "-c", \
-		"st -e /bin/sh -c \"curl -J -O --user-agent '$1'" \
+		"stterm -e /bin/sh -c \"curl -J -O --user-agent '$1'" \
 		" --referer '$2'" \
 		" -b ~/.surf/cookies.txt -c ~/.surf/cookies.txt '$0';" \
 		" sleep 5;\"", \
